src/main.cpp

```diff
@@ -556,18 +571,21 @@ Args parse_args(int argc, char* argv[])
       case 'c':
         args.cmd_str = optarg;
         break;
-      case '$':
+      case Options::USDT_SEMAPHORE:
         args.usdt_file_activation = true;
         break;
-      case 'u':
+      case Options::UNSAFE:
         args.safe_mode = false;
         break;
       case 'b':
+      case Options::BTF:
         break;
       case 'h':
+      case Options::HELP:
         usage();
         exit(0);
       case 'V':
+      case Options::VERSION:
```

Comment:
I wasn't sure about keeping these three unchanged vs using the enum. I went with using the enum for consistency, but happy to revert if folks think that makes things more complicated.

Reply:
I prefer consistency so I'm fine with the current solution.